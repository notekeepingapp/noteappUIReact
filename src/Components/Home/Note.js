import React from 'react';
import Card from "react-bootstrap/Card";
import Button from "react-bootstrap/Button";

const Note = (props) => {
    return (
        <div className={"note"} style={{
            width: 250,
            height: 250, overflow: 'auto'
        }}>
            <Card>
                <Button variant={"light"} onClick={()=>{props.handleNoteDelete(props.note)}} style={{width:"fit-content", alignSelf:"flex-end"}}>Delete</Button>
                <div onClick={() => {
                    props.handleCardClick(props.note)
                }}>
                    <Card.Body>
                        <Card.Title>{props.note.noteTitle}</Card.Title>
                        <Card.Text>{props.note.noteContent}</Card.Text>
                    </Card.Body>
                </div>
            </Card>
        </div>
    )
};

export default Note;