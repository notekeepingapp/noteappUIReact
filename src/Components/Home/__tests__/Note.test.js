import React from 'react';
import { shallow } from 'enzyme';
import Note from '../Note';

describe('Note View rendering', () => {
  test('should match Note view snapshot', () => {
    const note = {
      noteTitle: 'Anju',
      noteContent: 'admin1',
    };
    const component = shallow(<Note note={note} />);
    expect(component).toMatchSnapshot();
  });
});
