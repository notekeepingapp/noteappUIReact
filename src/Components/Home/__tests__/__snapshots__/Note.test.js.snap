// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note View rendering should match Note view snapshot 1`] = `ShallowWrapper {}`;
